### PR TITLE
Custom mark highlighting

### LINKED ISSUE #113: mark highlighting not compatible with new process for ordering marks
Hovering removes custom mark highlighting. 

# Test notes
* Add a custom mark via the text settings input:
```
{
    "custom_marks": [
        {
            "per": [
                "USUBJID",
                "VISIT",
                "TEST",
                "STRESN"
            ],
            "type": "circle",
            "attributes": {
                "fill": "red",
                "fill-opacity": 1,
                "stroke": "black",
                "stroke-opacity": 1
            },
            "radius": 4,
            "values": {
                "DY": [
                    "56",
                    "112",
                    "168",
                    "224",
                    "280",
                    "336"
                ]
            }
        }
    ]
}
```
* Verify that custom marks as well as default marks highlight as expected.
* Verify that custom marks as well as default marks can be clicked.
* Verify that custom marks maintain their visual characteristics, e.g. fill, stroke, etc.

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #113

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
     "webcharts": "^1.10"
   },
   "scripts": {
-    "build": "npm audit fix && npm run bundle && npm run format && npm run build-wki && npm run check-settings-schema",
+    "build": "npm audit fix && npm run bundle && npm run format && npm run build-wiki && npm run check-settings-schema",
     "build-wiki": "npm run build-configuration-wiki && npm run build-data-guidelines-wiki",
     "build-configuration-wiki": "node ./scripts/build-configuration-wiki.js",
-    "build-data-guidelines-wiki": "node ./scripts/build-data-guidlines-wiki.js",
+    "build-data-guidelines-wiki": "node ./scripts/build-data-guidelines-wiki.js",
```
Comment (pburnsdata) [inline package.json]: nice catch

File: scripts/check-settings-schema.js [hunk 2]
```diff
@@ -24,3 +24,6 @@ if (missingActualSettings.length > 0) {
     console.log('\x1b[31m%s\x1b[0m', 'These settings are missing from the renderer settings:\n');
     console.log('\x1b[31m%s\x1b[0m', `${JSON.stringify(missingActualSettings, null, 4)}\n`);
 }
+
+if (missingExpectedSettings.length === 0 && missingActualSettings.length === 0)
+    console.log('The settings schema and the renderer settings are *NSYNC!');
```
Comment (pburnsdata) [inline scripts/check-settings-schema.js]: .....

File: scripts/build-configuration-wiki.js [hunk 3]
```diff
@@ -3,9 +3,8 @@ const fs = require('fs');
 const pkg = require('../package');
 const schema = require('../settings-schema');
 const properties = schema.properties;
-const settings = require('../src/configuration/rendererSettings.js').default;
-const chartSettings = require('../src/configuration/chartSettings.js').default;
-const listingSettings = require('../src/configuration/listingSettings.js').default;
+const rendererSettings = require('../src/configuration/rendererSettings.js').default;
+const webchartsSettings = require('../src/configuration/webchartsSettings.js').default();
```
Comment (pburnsdata) [inline scripts/build-configuration-wiki.js]: is default a method?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Saftey Outlier Explorer
+# Safety Outlier Explorer
```
Comment (pburnsdata) [inline README.md]: MY MAN!!! YES!

File: scripts/build-configuration-wiki.js (hunk 3 above)
Comment (samussiah) [reply scripts/build-configuration-wiki.js]: i think that's how babel interprets import/export syntax in the context of require().  in this case the settings files return a default function.

**Pull request merged**